### PR TITLE
Migrate MySQL DB on postinst when upgrading

### PR DESCRIPTION
**Pull request by jacalvo**

### CODE REVIEW

File: debian/sogo.postinst [hunk 1]
```diff
@@ -14,6 +14,36 @@ set -e
...
+    sqlscript="$sqlscript$part1$part2"
+    IFS="$oldIFS"
+}
+
+function runDBMigration() {
+    username=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d: -f2|cut -d/ -f3)
+    password=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d: -f3|cut -d@ -f1)
+    hostname=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d@ -f2|cut -d: -f1)
+    database=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d/ -f4)
+    indextable=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}' |  awk -F/ '{print $NF}')
```
Comment (sixstone-qq) [inline debian/sogo.postinst]: Is it `awk` installed by default in the system? Or should we add it as dependency in sogo package?

Comment (jacalvo) [inline debian/sogo.postinst]: It's a dependency of the ubuntu-minimal metapackage, not sure if that's enough...

Comment (papajulio) [inline debian/sogo.postinst]: Everything should be right regarding awk http://unix.stackexchange.com/questions/82409/are-there-versions-of-unix-that-dont-have-awk-in-default-install

**Comment by papajulio** [comment]
:+1: Tested and working

File: debian/sogo.postinst [hunk 2]
```diff
@@ -14,6 +14,36 @@ set -e
 # the debian-policy package
 #
 
+# Migration code adapted from original interactive script: sql-update-2.1.17_to_2.3.0-mysql.sh
+
+sqlscript=""
+
+function adjustSchema() {
+    oldIFS="$IFS"
```
Comment (blaxter) [inline debian/sogo.postinst]: why is this needed?
